components/webui/imports/api/ingestion/server/CompressionDbManager.js

```diff
@@ -59,7 +59,9 @@ class CompressionDbManager {
                 (
                     SELECT *
                     FROM SelectedColumns
-                    WHERE _id=${jobId}
+                    WHERE 
+                        _id=${jobId} && 
```

Comment:
Can we use the `AND` operator which is less MySQL specific?

Reply:
This may no longer apply if you agree with the function body change proposed above.